src/lambda_streams/Numbers.java

```diff
@@ -11,81 +11,248 @@ public static void main(String[] args) {
         System.out.println(nums);
 
         //Part II - refactor all of the class methods to accept lambda expressions. You can put the lambda functions directly inside the method calls, or defined them first, then pass them into the methods. give them the same names as the static methods, but add the word 'lambda' in front of every lambda function:
-        /* e.g.
+//
+        Calulate LambdaSumVar = (nums) -> {
+            int sum = 0;
+            for (int i = 0; i < nums.size(); i++) {
+                      sum = sum + nums.get(i);
+               }
```

Comment:
It's interesting in this code that a lambda is created to return the sum of the list of numbers, while processing each number using a 'For' loop.  Consider at least using an Enhanced 'For' or 'ForEach' to process the list, possibly even implementing a lambda statement here to process the list of numbers as a stream.  I would make the same comment for the remainder of the Lambda implementations.